Userland/Libraries/LibJS/Runtime/VM.h

```diff
@@ -91,6 +91,14 @@ class VM : public RefCounted<VM> {
         return *m_single_ascii_character_strings[character];
     }
 
+    enum class ErrorMessage {
+        OutOfMemory,
+
+        // Keep this last:
+        Count,
```

Comment:
The canonical name for this is `__Count` (~55 definitions across the codebase according to a quick grep). Maybe while we're here let's also make is a bit more clear why only a selected subset of errors is present in this list (`pre_allocated_error_messages`, maybe?)

Reply:
er...`__Count` is UB, no? Double underscore identifiers are reserved. I'm not actually sure if that applies to enum values, but I've always argued to avoid the beast altogether and don't use double underscore anywhere
> Each name that contains a double underscore (__) or begins with an underscore followed by an uppercase letter (2.11) is reserved to the implementation for any use.

(I'll add a comment though)

Reply:
Hmm, we should probably avoid it then, but it doesn't seem to have caused any issues over the years either. Let's go with `__Count` for now and then :hypeshed: switching to `_Count` globally.

Reply:
When we get to that, FYI `_Count` is also UB: "or begins with an underscore followed by an uppercase letter"

IMO just "Count" or "Last" are rather standard enough in the industry, but we can bikeshed that when we get there